CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [0.5.3] - 2022-07-05
```

Comment:
When filling out the changelog in a PR, please do not specify a specific version. Instead, put your changes under an `Unreleased` heading. Changes that you make in a PR might not necessarily come out in the next version, or the next version might not be what you expect.